modules/lang-painless/src/main/java/org/elasticsearch/painless/symbol/ScriptScope.java

```diff
@@ -66,6 +66,8 @@ public ScriptScope(PainlessLookup painlessLookup, CompilerSettings compilerSetti
         staticConstants.put("$SOURCE", scriptSource);
         staticConstants.put("$DEFINITION", painlessLookup);
         staticConstants.put("$FUNCTIONS", functionTable);
+        // TODO(stu): inject compiler settings here
```

Comment:
leftover todo?

Reply:
Yup, these and all the left over todos were why this was initially a draft.  They are gone now.